version/version.go

```diff
@@ -6,35 +6,44 @@ import (
 	"runtime"
 )
 
-// Variables set by build flags
+// Variables representin application's versioning
+// information set at build time.
 var (
-	Commit    = ""
-	Version   = ""
+	// Application's name
+	Name = ""
```

Comment:
This PR should probably show an example for how the `gaia` Name is set here (within the `cmd/gaia`) - I'm assuming you just forgot this

Reply:
> I'm assuming you just forgot this

I did not: https://github.com/cosmos/cosmos-sdk/pull/4250/files#diff-b67911656ef5d18c4ae36cb6741b7965R47

All these variables are set via build flags. I didn't provide examples as there were any before.

Reply:
I've improved the version package docs